src/internal/clusterstate/v2.8.0_test.go

```diff
@@ -29,6 +33,40 @@ func Test_v2_8_0_ClusterState(t *testing.T) {
 	require.NoError(t, migrations.ApplyMigrations(ctx, db, migrationEnv, state_2_8_0))
 	require.NoError(t, migrations.BlockUntil(ctx, db, state_2_8_0))
 
+	// Get all collections commits.
+	expectedCommits, expectedAncestries, err := v2_8_0.ListCommitsFromCollection(ctx, db)
+	require.NoError(t, err, "should be able to list commits from collection")
+
+	// Get all pfs.commits.
+	gotAncestries := make(map[string]string)
+	iter, err := pfsdb.ListCommit(ctx, db, nil, false)
+	require.NoError(t, err, "should be able to list commits from pfs.commits")
+	var gotCommits []v2_8_0.CommitInfo
+	require.NoError(t, stream.ForEach[pfsdb.CommitPair](ctx, iter, func(commitPair pfsdb.CommitPair) error {
```

Comment:
It's debatable whether we want to use these APIs here. On one hand, it's super convenient, but on the other hand, migration code should not depend on code that could change. Especially this list commits functionality, which might change in the future. In the previous migrations, we simply do a `SelectContext` and scan the data into a slice of db model structs.